resources/views/plotchoose.blade.php

```diff
@@ -1,98 +1,109 @@
+  <link rel="icon" type="image/png" sizes="32x32" href="{{ asset('/assets/favicon/favicon-32x32.png') }}">
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('/assets/favicon/favicon-16x16.png') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/navbar-ex.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/footer.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/login-caas-menu.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/about.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/home.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/form-style.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/list.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/main.css') }}">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Electrolize&family=Fira+Code:wght@400&display=swap" rel="stylesheet">
 </head>
-<body id="list-section">
 
-@include('layouts/navbarCaasLogout')
-@section('navcaaslogout')
-@endsection
+<body style="background-color: #2c4d16">
+
+  @include('layouts/navbarCaasLogout')
+  @section('navcaaslogout')
+  @endsection
 
-<section id="list-section">
-<div class="container pb-5">
-    <div class="pt-5 d-flex justify-content-center">
-        <div style="background-color: #FFB936" class="checker-box">
+  <section id="landing-section">
+    <div class="container pb-5">
+      <div class="pt-5 d-flex justify-content-center">
+        <div class="checker-box">
           <div class="text-center text-nim-head">
             <span>PILIH JADWAL REKRUTMEN</span>
           </div>
-          <div class="text-center pt-3 pb-3">
+          <div class="text-center pt-3 pb-3 rekrutmen-warning">
             <div class="text-center">
-              <span style="color: rgb(255, 43, 43);font-size:1.5rem;font-weight:700;">!!!PERHATIAN!!!</span>
+              <span class= "warning-loop" style="color: rgb(255, 43, 43)">!!!PERHATIAN!!!</span>
             </div>
-            <div style="background-color: #de960e;border-radius: 0.7rem;margin-top:0.6rem;">
-                <span style="color: rgb(228, 228, 228);font-size:1.5rem;font-weight:700;">Setelah pilih jadwal, kalian tidak bisa ganti jadwal lagi</span>
+            <div style="background-color: #57773f;border-radius: 0.7rem;margin-top:0.6rem;">
+              <span class="mention" style="color: rgb(228, 228, 228)">Setelah pilih jadwal, kalian tidak bisa ganti jadwal lagi</span>
```

Comment:
```suggestion
            <div style="background-color: #57773f;border-radius: 0.7rem;margin-top:0.6rem;" class="pt-2">
              <span class="mention" style="color: rgb(228, 228, 228)">Setelah pilih jadwal, kalian tidak bisa ganti jadwal lagi</span>
```

Reply:
coba tambahin `pt-2` dip, itu terlalu mepet

Reply:
Ini bang untuk `pt-2`:

![image](https://user-images.githubusercontent.com/88444330/160150177-440d6f0d-a8d0-453a-8837-2d93f14c55c7.png)

mungkin yang diganti 'margin-top : 0.6rem' jadi 'margin-top : 1rem' bang :
```suggestion
             <div style="background-color: #57773f;border-radius: 0.7rem;margin-top:1rem;">
              <span class="mention" style="color: rgb(228, 228, 228)">Setelah pilih jadwal, kalian tidak bisa ganti jadwal lagi</span>
```
![image](https://user-images.githubusercontent.com/88444330/160152110-15b4a408-1885-4d9b-a253-3f55af9d9b90.png)

Reply:
nah iya dip margin maksud saya, oke mantapp wkwkwk

Reply:
siap bang